### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by Drocheg**
Adds Travis integration

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -40,6 +40,7 @@ group :development, :test do
   gem 'capybara', '~> 2.13'
   gem 'selenium-webdriver'
   gem 'rubocop', '0.58.2'
+  gem 'rspec'
```
Comment (Drocheg) [inline Gemfile]: Me rompia rspec entonces agregué la gema

Comment (Gmarn) [reply Gemfile]: :+1: but you need rspec only on your `test` environment.

**Review by Gmarn — CHANGES_REQUESTED** [review]
I will just add you a task. I saw you are using ruby 2.5.1 in your porject. 
By convention a rails app have on the root a `ruby-version` file with that indicate the version is using, so please try to add it ;)

File: app/channels/application_cable/channel.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (Gmarn) [inline app/channels/application_cable/channel.rb:1]: :clap: I think rubocop ask you for that.
Here (in Wolox), we are still not using that but in the futur we will implement it and needed for ruby 3. Just for the info ;)

**Review by Gmarn — APPROVED** [review]

**Pull request merged**